### ISSUE TITLE
can't set DNS2 to empty or no

### ISSUE THREAD

**Issue by sean-abbott** [MoJo2600/pihole-kubernetes#68]
docker-pi-hole configuration allows you to set DNS2 to no if you want a single dns server (which I do, because I'm running coredns on k8s with the pi, and that should be my only dns)

When I set DNS2 to "no" in values, it fails the checks.

When I set it to no (no quotation marks) it passes through, but then sets google dns as the second dns.

I'm new to k8s and haven't figured this out yet.

https://github.com/pi-hole/docker-pi-hole/blob/master/README.md#environment-variables

**Comment by sean-abbott** [comment]
(and then I can temporarily change this in active esttings, but eventually it picks it back up from the configmap.)

**Comment by SiM22** [comment]
They should be hardcoded by default in `templates/deployment.yaml`

```
          - name: 'DNS1'
            value: "127.0.0.1#5053"
          - name: 'DNS2'
            value: "127.0.0.1#5053"
```

**Comment by SiM22** [comment]
@MoJo2600 i think you can close this

**Comment by sean-abbott** [comment]
I mean I don't think you should have to go into the template to set this;
Plus, it's defaulting to  either google or cloudflare (can't recall), not
localhost, so that doesn't seem like the problem.

**Comment by MoJo2600** [comment]
I think the `dnsConfig` setting might be the issue? It sets the default DNS to google. 

```
      dnsConfig:
        nameservers:
        - 127.0.0.1
        - 8.8.8.8
```

But i can't figure out, why setting `DNS2` to `no` should not work.

**Comment by Ornias1993** [comment]
"no" is not an option.

Firstoff: Even if it was a boolean (true or false question) it needed to be `false` not `no`
Second: it is not a boolean it's a string, empty is the only option for it to not be added in the template.

**Comment by MoJo2600** [comment]
The documentation states:

Secondary upstream DNS provider, default is google DNS, `no` if only one DNS should used

So I'd say "no" as a string value is correct if I interpret the documentation right.

**Comment by Ornias1993** [comment]
Ahh shoot yes, you're right @MoJo2600 it's procesed on their side.
I was assuming it was processed on yours :)

That being said: It's depricated anyway, so maybe beter just replace it with the new env while at it?

**Comment by MoJo2600** [comment]
No worries. Would make absolute sense to replace it. Maybe somebody finds the time to create a pull request 😉

**Comment by MoJo2600** [comment]
Fixed with #183
